hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -105,7 +105,7 @@ public void testCreateTableBuilder() throws Exception {
     Schema schema = getTestSchema();
     PartitionSpec spec = PartitionSpec.builderFor(schema).bucket("data", 16).build();
     TableIdentifier tableIdent = TableIdentifier.of(DB_NAME, "tbl");
-    String location = temp.newFolder("tbl").toString();
+    String location = temp.resolve("tbl").toString();
```

Comment:
does this produce the same result as the old code?

Reply:
Yes, it creates a new folder